config/locales/active_admin.de.yml

```diff
@@ -431,6 +432,12 @@ de:
           notice: "Rückgängig:"
         action_item:
           link_to: "rückgängig"
+      member_action:
+        set_menu_item_online: "Diesen Menüpunkt online schalten."
```

Comment:
No it was not, i took the same wording like for articles. It could be a new Task to rethink wordings at all!?